rhizome/host/bin/setup-vm

```diff
@@ -0,0 +1,70 @@
+#!/bin/env ruby
+# frozen_string_literal: true
+
+require "json"
+require_relative "../lib/vm_setup"
+
+unless (verb = ARGV.shift)
```

Comment:
The `verb` seems quite unusual to me. Additionally, the error message is `"expected action as argument"`. Should we rename the variable to `action`?